app/presenters/publishing_api/ministers_index_presenter.rb

```diff
@@ -100,5 +105,16 @@ def ordered_ministerial_departments
         .uniq
         .map(&:content_id)
     end
+
+    def ordered_whips(whip_type)
```

Comment:
This method returns content ids, so perhaps `ordered_whip_content_ids` or something would be a better name?